modules/storage-api/src/test/java/org/apache/ignite/internal/storage/AbstractMvTableStorageTest.java

```diff
@@ -70,38 +85,65 @@ void setUp() {
     }
 
     @AfterEach
-    void tearDown() {
-        tableStorage.destroy();
+    void tearDownBase() {
+        tableStorage.stop();
     }
 
     /**
-     * Tests creating a partition.
+     * Tests that {@link MvTableStorage#getMvPartition(int)} correctly returns an existing partition.
      */
     @Test
-    public void testCreatePartition() {
-        int partitionId = 0;
+    void testCreatePartition() {
+        MvPartitionStorage absentStorage = tableStorage.getMvPartition(0);
+
+        assertThat(absentStorage, is(nullValue()));
+
+        MvPartitionStorage partitionStorage = tableStorage.getOrCreateMvPartition(0);
+
+        assertThat(partitionStorage, is(notNullValue()));
+
+        var testData = binaryRow(new TestKey(1, "1"), new TestValue(10, "10"));
```

Comment:
illegal `var`

Reply:
This code was copied from other class. Will fix